web/admin/src/App.js

```diff
@@ -190,6 +203,20 @@ export default class App extends Component {
     }
   }
 
+  unBlock = (reports, key, userId) => {
+    if (reports.length) {
+      reports.map((item) => {
```

Comment:
`forEach` is more idiomatic than `map` here, even though `map` will work.  Typically, `forEach` is used to "do" something for each item, while `map` is for transforming an array of objects of one type into an array of objects of another type; however, we are not returning anything from the function passed to `map`.  We are only taking an action, so `forEach` more accurately describes what is happening here.